src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -115,6 +115,22 @@ public ConfigurationResponse createConfiguration(String vmName, String image, St
         return response;
     }
 
+    public ConfigurationResponse createConfiguration(String vmName, String image, String baseImage,
+            String configTemplate, int cpuCount, String scheduler) throws IOException {
+        Gson gson = new Gson();
```

Comment:
Can we make the old overload call the new one? There is no need to duplicate the code